### DISCUSSION TITLE
new kubernetes install via helm, support for block storage?

### DISCUSSION THREAD

**Discussion by webdz9r** [cortexproject/cortex#3691] [Q&A]
New to cortex, have it working from the Getting Started documentation, I'm working on deploying this in our Kube infra and I'm not seeing that Block storage is supported, seems you can define blocks as the storage type but there isn't an example for setting S3 bucket details. 

TLDR; Can we use blocks storage with Kubernetes / Helm installation? Any example configurations?

**Comment by jtlisi** [comment]
> TLDR; Can we use blocks storage with Kubernetes / Helm installation? Any example configurations?

The [existing helm chart](https://github.com/cortexproject/cortex-helm-chart) does not include support for blocks storage. However, it would not be difficult to modify the chart to add a blocks storage config. No one is actively working on the helm chart to allow it support Blocks Storage. But since chunks is in maintenance mode, we will likely upgrade the published charts in the near future.

You can track the progress here: #2424
